src/errors/validation_exception.rs

```diff
@@ -129,13 +132,15 @@ impl ValidationError {
         title: PyObject,
         line_errors: &PyList,
         error_mode: Option<&str>,
+        hide_input: Option<bool>,
```

Comment:
```suggestion
        hide_input: bool,
```
Then add above this fn definition

```rs
#[pyo3(signature = (title, line_errors, error_mode=None, hide_input=false))]
```